exporters/trace/gcp_exporter/recordable.h

```diff
@@ -22,16 +22,26 @@ class Recordable final : public sdk::trace::Recordable
 public:
   const google::devtools::cloudtrace::v2::Span &span() const noexcept { return span_; }
 
-  void SetIds(trace::TraceId trace_id,
-              trace::SpanId span_id,
-              trace::SpanId parent_span_id) noexcept override;
+  void SetIds(opentelemetry::trace::TraceId trace_id,
+                      opentelemetry::trace::SpanId span_id,
+                      opentelemetry::trace::SpanId parent_span_id) noexcept override;
 
   void SetAttribute(nostd::string_view key,
-                    const opentelemetry::common::AttributeValue &&value) noexcept override;
+                            const opentelemetry::common::AttributeValue &value) noexcept override;
 
-  void AddEvent(nostd::string_view name, core::SystemTimestamp timestamp) noexcept override;
+  void AddEvent(
+      nostd::string_view name,
+      core::SystemTimestamp timestamp = core::SystemTimestamp(std::chrono::system_clock::now()),
```

Comment:
I don't think these defaults are necessary. Once https://github.com/open-telemetry/opentelemetry-cpp/pull/208 merges, all defaults will be handled using function overloading in the base `Recordable` class. The same applies to `AddLink()`.

Reply:
Oh I see.
Alright. Thanks!

Reply:
I am assuming I'll have to wait for it to be merged and then run my tests?